go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/apache/pulsar-client-go v0.1.0
 	github.com/aws/aws-sdk-go v1.25.0
 	github.com/baiyubin/aliyun-sts-go-sdk v0.0.0-20180326062324-cfa1a18b161f // indirect
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869
```

Comment:
Can you use `"github.com/stretchr/testify/assert"`? this is the assertion package we use through-out Dapr.

Reply:
Hmm. My bad, must have been an autocomplete fail.